src/layouts/general/layout.module.css

```diff
@@ -145,33 +243,66 @@
   color: var(--color-primary);
 }
 .nav__inner ul {
-  list-style-type: lower-roman;
-  padding-left: 1.5rem;
+  list-style-type: none;
+  padding-left: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
 .nav__inner__link {
+  flex: 1;
+  width: 20rem;
+  padding-top: 2rem;
   padding-bottom: 1rem;
   color: var(--color-secondary3__shade1);
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.nav__inner__link a {
+  color: var(--color-standard__white);
+  font-size: 1.75rem;
 }
 
 .nav__inner__link__active {
   padding-bottom: 1rem;
+  padding-top: 2rem;
   color: var(--color-primary__tint1);
+  flex: 1;
+  width: 20rem;
+  align-items: flex-start;
+  font-size: 1.75rem;
 }
 .nav__inner__link__active a {
   border-bottom: 2px solid var(--color-primary__tint2);
+  color: var(--color-standard__white);
+}
+
+.nav__inner__link__child {
+  display: none;
+}
+.nav__inner__link__children a {
+  color: var(--color-standard__white);
+  font-size: 1.5rem;
+  margin-left: 1rem;
+}
```

Comment:
```suggestion

.nav__inner__link__child {
  display: none;
  margin-left: 1rem;
  margin-bottom: 1rem;
}
.nav__inner__link__children a {
  color: var(--color-standard__white);
  font-size: 1.5rem;
}
```

Kan gjøres for å få underpunktene på siden til å ha samme margin

<img width="184" alt="Screenshot 2023-01-03 at 13 02 40" src="https://user-images.githubusercontent.com/110395932/210353646-afecf00d-8c62-425c-bbdb-f9f5a79e8bbc.png">

Reply:
Topp! Jeg har implementert det rett i koden, så det er ikke nødvendig å committe.